tests/test_cookiecutter_template.py

```diff
@@ -0,0 +1,59 @@
+# pylint: disable=redefined-outer-name
+# standard library imports
+import pathlib
+import subprocess
+import uuid
+
+# third party imports
+import pytest
+from cookiecutter.main import cookiecutter
+
+
+# local imports
+from utility.find_package_manager import get_package_manager_executable
+
+
+TEMPLATE_DIRECTORY = str(pathlib.Path(__file__).parent.parent)
+EXECUTABLE = get_package_manager_executable()
+
+
+@pytest.fixture
+def default_template(tmp_path):
+    env_name_id = f"pytest_{uuid.uuid4()}-env"
+    cookiecutter(
```

Comment:
Am I right to assume that this initializes the repo an therefore runs pre-commit checks on the initialized repo?

Reply:
yes